### PR TITLE
Added atlassian backup code version 1.0

### PR DESCRIPTION
**Pull request by vchokshi**
This is the Proof of Concept code for using Elastio to back up Atlassian.

Successful run below:
```
(ab) backup atlassian_backup --use-ssm                                                                                                                      
Backing up Confluence without attachments.                                                                                                                    
Backing up Jira without attachments.                                                                                                                          
Downloading jira.zip for jira from https://theonlyiot4.atlassian.net/plugins/servlet/export/download/?fileId=613e41e5-0da8-4ae4-81b8-9d01a2126edb to /tmp/    
Downloading confluence.zip for confluence from https://theonlyiot4.atlassian.net/wiki/download/temp/filestore/b7e20910-5085-4d20-98c6-5e98d475cc88 to /tmp/   
Protecting /tmp/jira.zip using Elastio...                                                                                                                     
Backing up /tmp/jira.zip source paths to vihar vault using variable shard                                                                                     
▪ Total               [00:00:03] ############################################################ 101.50KB/101.00KB Succeeded                                     
▪   /tmp/jira.zip     [00:00:03] ############################################################ 101.50KB/101.00KB Succeeded                                     
Ingested 101.50KB in 1 item(s) in 3 seconds (32.93KB/s, 1.93MB/m)                                                                                             
New: 101.50KB (32.93KB/s, 1.93MB/m)                                                                                                                           
Existing: 0B (0B/s, 0B/m)                                                                                                                                     
Skipped: 0B (0B/s, 0B/m)                                                                                                                                      
Recovery point ID: rp-01fzgqwr9s4rsrq1ffqr0y02nx                                                                                                              
Protecting /tmp/confluence.zip using Elastio...                                                                                                               
Backing up /tmp/confluence.zip source paths to vihar vault using variable shard                                                                               
▪ Total               [00:00:03] ############################################################ 49.50KB/49.00KB Succeeded                                       
▪   /tmp/confluence.zi[00:00:03] ############################################################ 49.50KB/49.00KB Succeeded                                       
Ingested 49.50KB in 1 item(s) in 3 seconds (12.90KB/s, 774.20KB/m)                                                                                            
New: 49.50KB (12.90KB/s, 774.20KB/m)                                                                                                                          
Existing: 0B (0B/s, 0B/m)                                                                                                                                     
Skipped: 0B (0B/s, 0B/m)                                                                                                                                      
Recovery point ID: rp-01fzgqx8qvtvmv3khgy15kenzf
```

### CODE REVIEW

File: atlassian_backup-1.0.0/vendor/confluence.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Open Source code imported for use by author. Use make to update
```
Comment (anelson) [inline atlassian_backup-1.0.0/vendor/confluence.py:1]: What was the license on this code?

File: atlassian_backup-1.0.0/vendor/jira.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Open Source code imported for use by author. Use make to update
```
Comment (anelson) [inline atlassian_backup-1.0.0/vendor/jira.py:1]: Similar question regarding license.

File: atlassian_backup-1.0.0/PKG-INFO [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Metadata-Version: 1.2
+Name: atlassian_backup
+Version: 1.0.0
+Summary: Backup Atlassian Using Elastio
+Home-page: UNKNOWN
+Author: Vihar Chokshi
+Author-email: vc@iot4.net
+License: UNKNOWN
```
Comment (anelson) [inline atlassian_backup-1.0.0/PKG-INFO:8]: Let's release this under MIT and Apache 2, unless the `confluence` and `jira` modules have an incompatible license in which case we have to use their license instead.

**Review by anelson — CHANGES_REQUESTED** [review]

File: atlassian_backup-1.0.0/vendor/confluence.py (hunk 1 above)
Comment (vchokshi) [reply atlassian_backup-1.0.0/vendor/confluence.py:1]: Thanks for asking Adam. 

https://bitbucket.org/atlassianlabs/automatic-cloud-backup/src/master/ is the source of the original code. 

On PR inspection, I discovered the Makefile used to track the commit sha of this "pseudo fork" from atlassian labs was not tracked. I've added a MANIFEST.in file to ensure the original Makefile used to pull this code is included in the sdist package.

Comment (vchokshi) [reply atlassian_backup-1.0.0/vendor/confluence.py:1]: Good catch, here is the Makefile and commit that tracks the "pseudo fork" of the code.

File: atlassian_backup-1.0.0/vendor/jira.py (hunk 2 above)
Comment (vchokshi) [reply atlassian_backup-1.0.0/vendor/jira.py:1]: Same answer as above. Atlassian Code is currently not licences so I assume it falls under Bitbucket Terms of Service.

**Review by anelson — APPROVED** [review]

**Pull request merged**